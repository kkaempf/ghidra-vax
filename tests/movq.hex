7d509fb2041420
