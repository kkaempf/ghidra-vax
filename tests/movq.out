0x00000000: MOVQ R0, @#0x201404b2
